nemo/core/nemo_model.py

```diff
@@ -0,0 +1,243 @@
+# ! /usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2019-, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import random
+import shutil
+import string
+import tarfile
+from abc import abstractmethod
+from os import path
+from typing import Iterable
+
+from .neural_factory import DeploymentFormat, OperationMode
+from .neural_graph import NeuralGraph
+from .neural_modules import ModuleType, NeuralModule
```

Comment:
Please use absolute imports not relative.

Reply:
fixed